src/contexts/ArticleContext/ArticleContext.tsx

```diff
@@ -0,0 +1,19 @@
+import { ArticleAPI } from '@/api/Article';
+import { createContext, useContext } from 'react';
+
+export interface ArticleContextProps {
+  articles: ArticleAPI[] | null;
```

Comment:
Please remove null

Reply:
If i remove null, i will face with some problem in page.tsx, because when articles are loading, Image with 'no image' will display together with Loader